src/main/java/com/minecolonies/blocks/ModBlocks.java

```diff
@@ -24,6 +24,10 @@
     public static final Block blockHutFarmer     = new BlockHutFarmer();
     public static final Block blockHutFisherman  = new BlockHutFisherman();
     public static final Block blockSubstitution  = new BlockSubstitution();
+    public static final Block blockHutField      = new BlockHutField();
+
+    // Deactivated for now
+    // public static final Block blockBarrel        = new BlockBarrel();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)